internal/v2/tls_config_store/tls_config_store.go

```diff
@@ -0,0 +1,114 @@
+// Static implementation of TLS Configuration Store (no calls to S2Av2, Remote Signer Library, Certificate Verifier)
```

Comment:
nit: I think the package comment should reflect what the library does (as opposed to how it does it, e.g. by using static creds).

To that end, I would suggest saying something like "Builds TLS configurations that offload operations to S2Av2."

Reply:
Will do!